src/webgpu/api/validation/encoding/cmds/setBindGroup.spec.ts

```diff
@@ -130,36 +176,41 @@ g.test('dynamic_offsets_match_expectations_in_pass_encoder')
         { dynamicOffsets: [0, 1024], _success: false },
         { dynamicOffsets: [0, 0xffffffff], _success: false },
       ])
+      .combine(pbool('useU32array'))
   )
   .fn(async t => {
     // Dynamic buffer offsets require offset to be divisible by 256
-    const MIN_DYNAMIC_BUFFER_OFFSET_ALIGNMENT = 256;
-    const BINDING_SIZE = 9;
+    const kMinDynamicBufferOffsetAlignment = 256;
```

Comment:
I think this could be kUniformBufferOffsetAlignment and be in capability_info.ts. I'm surprised we don't have a limit for this in the spec too.